lib/libutee/arch/arm/sub.mk

```diff
@@ -2,4 +2,5 @@ cppflags-y += -I$(sub-dir)/../..
 
 srcs-y += user_ta_entry.c
 srcs-y += utee_misc.c
-srcs-y += utee_syscalls_asm.S
+srcs-$(CFG_ARM32_user_ta) += utee_syscalls_a32.S
+srcs-$(CFG_ARM64_user_ta) += utee_syscalls_a64.S
```

Comment:
Could it be stated more explicitly in the commit message, or later in documentation, that libutee is compiled for a given 32bits or 64bits mode, but right now, we do not provide both versions of the library?